src/Symfony/Component/Security/Http/Authenticator/AccessToken/AccessTokenHandlerInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Http\Authenticator\AccessToken;
+
+use Symfony\Component\Security\Core\Exception\AuthenticationException;
+
+/**
+ * The token handler retrieves the user identifier using the token.
+ *
+ * @author Florent Morselli <florent.morselli@spomky-labs.com>
+ */
+interface AccessTokenHandlerInterface
+{
+    /**
+     * @throws AuthenticationException
+     */
+    public function getUserIdentifierFrom(string $token): string;
```

Comment:
I would renamed the argument to `$accessToken` (and also update the phpdoc of the inteface)

Reply:
OK. Now it's renamed.